packages/split-button/README.md

```diff
@@ -2,6 +2,10 @@
 
 An `sp-split-button` surfaces an immediately envokable action via it's main button, as well as a list of alternative actions in its toggleable menu overlay. By default, any actions envoked from the overlay will replace the main action button. When leveraging `[type="more"]` the action will be envoked, but the main button will remain the action initally persribed by the implementor.
 
+## Deprecated
```

Comment:
Sorry, I didn't mean that we needed new content (or in this case, duplicate), just that the Inline Alert that you've added should include an anchor link that we could address with `#deprecation` to ensure that the link shared in Dev Mode got people to the appropriate part of the page.